### PR TITLE
SOF-7261: upgrade mkdocs-material to v9.x

### PR DESCRIPTION
**Pull request by pranabdas**
Preview: https://deploy-preview-275--mat3ra-documentation.netlify.app/

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -32,17 +32,17 @@ extra:
...
+        - icon: fontawesome/brands/linkedin
           link: https://www.linkedin.com/company/mat3ra/
 # Doesn't work with our version. It works from version 7.1.8
 #    analytics:
 #        provider: google
 #        property: UA-69270713-5
 markdown_extensions:
-    - markdown_include.include:
-        base_path: data
+    # - markdown_include.include:
```
Comment (pranabdas) [inline mkdocs.yml:44]: `markdown_include` is not compatible, check which features are affected.

File: mkdocs.yml [hunk 2]
```diff
@@ -58,10 +60,13 @@ markdown_extensions:
     - pymdownx.details              # Allow admonition-type collapsible <details>
     - pymdownx.superfences          # Allow for code blocks "fencing"
     # https://python-markdown.github.io/extensions/extra/#markdown-inside-html-blocks
-    - pymdownx.extrarawhtml         # Enables parsing of markdown inside html tags
+    # - pymdownx.extrarawhtml         # Enables parsing of markdown inside html tags
```
Comment (pranabdas) [inline mkdocs.yml:63]: It seems `extrarawhtml` is now obsolete, going to replace it with `md_in_html`.

File: mkdocs.yml (hunk 1 above)
Comment (timurbazhirov) [reply mkdocs.yml:48]: Let's remove it then?

**Review by timurbazhirov — APPROVED** [review]

**Comment by pranabdas** [comment]
`\boldsymbol` is broken with mathjax v3. The problem is resolved if we use katex instead - https://deploy-preview-275--mat3ra-documentation.netlify.app/properties-directory/non-scalar/stress-tensor/

boldsymbol

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**